docs/conf.py

```diff
@@ -215,15 +215,24 @@
 
 # -- Options for the Sphinx gallery -------------------------------------------
 
-extensions += ["sphinx_gallery.gen_gallery"]
-
-sphinx_gallery_conf = {
-    'filename_pattern': '^((?!skip_).)*$', # execute all examples except those that start with "skip_"
-    'examples_dirs': '..{}examples'.format(os.sep), # path to the examples scripts
-    'gallery_dirs': 'auto_examples', # path to save gallery generated examples
-    'reference_url': {
-        'astropy': None,
-        'matplotlib': 'http://matplotlib.org/',
-        'numpy': 'http://docs.scipy.org/doc/numpy/',
+try:
+    import sphinx_gallery
+    extensions += ["sphinx_gallery.gen_gallery"]
+
+    sphinx_gallery_conf = {
+        'mod_example_dir': 'generated/modules', # path to store the module using example template
+        'filename_pattern': '^((?!skip_).)*$', # execute all examples except those that start with "skip_"
+        'examples_dirs': '..{}examples'.format(os.sep), # path to the examples scripts
+        'gallery_dirs': 'generated/examples', # path to save gallery generated examples
```

Comment:
I recently changed this for SunPy so that it results in a nice URL for the gallery: http://docs.sunpy.org/en/latest/gallery/ which is maybe something to think about.